Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs

```diff
@@ -71,11 +71,11 @@ public QuestionController(IQuestionRepository questionsRepository, UserManager<A
         /// API to get all the Questions
         /// </summary>
         /// <returns>Questions List</returns>
-        [HttpGet]
-        public async Task<IActionResult> GetAllQuestionsAsync()
+        [HttpGet("{numberOfCalls}/{categoryId}/{difficultyLevel}/{searchQuestion?}")]
```

Comment:
we can give a different name to numberOfCalls as we are not passing number that represents a number of calls but we are passing number that is last fetched id of a question